spring-social-zotero/src/main/java/org/springframework/social/zotero/api/impl/GroupCollectionsTemplate.java

```diff
@@ -199,4 +203,27 @@ private long getLatestVersion(HttpHeaders responseHeaders) {
         }
         return -1;
     }
+
+    @Override
+    public ItemCreationResponse createCollection(String groupId, String collectionName, String parentCollection) throws ZoteroConnectionException {
+        String url = String.format("groups/%s/collections", groupId);
+        
+        ObjectMapper mapper = new ObjectMapper();
+        
+        JsonNode dataAsJson = mapper.createObjectNode()
+                .put("name", collectionName)
+                .put("parentCollection", parentCollection);
+        
+        ArrayNode jsonArray = mapper.createArrayNode();
+        jsonArray.add(dataAsJson);
+        
+        HttpEntity<ArrayNode> data = new HttpEntity<ArrayNode>(jsonArray);
+        
+        try {
+            return restTemplate.exchange(buildUri(url, false), HttpMethod.POST, data, ItemCreationResponse.class)
```

Comment:
this should use the buildGroupUri method or is there a reason it is not used here?